src/main/frontend/handler/editor.cljs

```diff
@@ -2542,7 +2540,7 @@
         (let [selection (get-selection-and-format)
               {:keys [selection-start selection-end value]} selection]
           (if (not= selection-start selection-end)
-            (do (delete-and-update selection-start selection-end)
+            (do (delete-and-update input selection-start selection-end)
```

Comment:
This caught a bug with incorrect args being passed

Reply:
👍, This error is addressed in PR https://github.com/logseq/logseq/pull/3864.